### ISSUE TITLE
Removing a user from a team

### ISSUE THREAD

**Issue by theoprata** [RocketChat/Rocket.Chat#23926]
### Description:

After upgrading to 4.2 version, removing a user from a team gives an "invalid-param" error.

### Steps to reproduce:

1. Add a user to a team.
2. Try removing the newly added user from the team.
3. 

### Expected behavior:

It would remove the user from the team.

### Actual behavior:

![image](https://user-images.githubusercontent.com/84879969/145642326-9aa5c2eb-95cb-4318-bc85-beb4b2efb215.png)
![image](https://user-images.githubusercontent.com/84879969/145642433-94d3a00c-5e1c-44a0-9806-9ab3fd5807b0.png)

### Server Setup Information:

- Version of Rocket.Chat Server: 
- Operating System: Ubuntu 20
- Deployment Method:  Docker
- Number of Running Instances: 1
- DB Replicaset Oplog: True
- NodeJS Version: v12.22.1
- MongoDB Version: 4.2.17

### Client Setup Information

- Desktop App or Browser Version: Any
- Operating System: Any

### Relevant logs:

```
rocketchat_1          | {"level":35,"time":"2021-12-10T21:18:20.147Z","pid":10,"hostname":"73b3239f50c0","name":"API","method":"POST","url":"/api/v1/method.call/license%3AgetTags","userId":"Rz6wtrA4HNCNy2v4t","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:95.0) Gecko/20100101 Firefox/95.0","length":"93","host":"localhost:3000","referer":"http://localhost:3000/admin/info","remoteIP":"172.20.0.1","status":200,"responseTime":1}
rocketchat_1          | {"level":35,"time":"2021-12-10T21:18:20.161Z","pid":10,"hostname":"73b3239f50c0","name":"API","method":"GET","url":"/api/v1/licenses.get","userId":"Rz6wtrA4HNCNy2v4t","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:95.0) Gecko/20100101 Firefox/95.0","host":"localhost:3000","referer":"http://localhost:3000/admin/info","remoteIP":"172.20.0.1","status":200,"responseTime":3}
rocketchat_1          | {"level":35,"time":"2021-12-10T21:18:20.189Z","pid":10,"hostname":"73b3239f50c0","name":"API","method":"GET","url":"/api/v1/licenses.maxActiveUsers","userId":"Rz6wtrA4HNCNy2v4t","userAgent":"Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:95.0) Gecko/20100101 Firefox/95.0","host":"localhost:3000","referer":"http://localhost:3000/admin/info","remoteIP":"172.20.0.1","status":200,"responseTime":7}

```

**Comment by yash-rajpal** [comment]
I believe this is fixed by #23857 and the fix is already released in 4.2.1 update. Please check if the issue still persists on 4.2.1

**Comment by theoprata** [comment]
Confirmed; issue has been resolved. Thank you!
